artiq/coredevice/trf372017.py

```diff
@@ -84,6 +84,9 @@ def __init__(self, updates=None):
             setattr(self, key, value)
 
     def get_mmap(self):
+        """Memory map for TRF372017
+
+        VCO calibration, and RF/LO outputs should be initially disabled"""
```

Comment:
That's a "must" now. And it's unclear what the user needs to do to enable them.

Reply:
I'm not aware of a requirement to do so. It is against the manufacturer recommendation and may lead to trouble with the calibration in marginal cases. The PR adds methods in the phaser driver to enable these RF/LO outputs and VCO calibration. It seems reasonable for users to look through the phaser documentation/source to find these. Most users will also not need to touch this. The VCO is calibrated and the TRF RF output is enabled in `init()`.

Reply:
If the user enables calibration initially, the calibration that you implement will break, no?

Still, if a user sees "RF output should initially be disabled" they ask themselves what they need to do to (a) heed that advice and (b) enable RF out.

Reply:
> If the user enables calibration initially, the calibration that you implement will break, no?

It is unspecified behaviour


> "RF output should initially be disabled" they ask themselves what they need to do to (a) heed that advice and (b) enable RF out.

Ack, I'll add comments to the bit entries instead.